static/js/shared/types.ts

```diff
@@ -145,3 +145,15 @@ export interface LinkedNodes {
 export interface GraphNodes {
   nodes: GraphNode[];
 }
+
+export interface SvgSearchResult {
```

Comment:
Nit: can we document which API call these are from?

Reply:
done